src/parsers/manifestjson.js

```diff
@@ -806,6 +811,15 @@ export default class ManifestJSONParser extends JSONParser {
     }
   }
 
+  validateHomePageURL(url) {
+    RESTRICTED_HOMEPAGE_URLS.forEach((domain) => {
+      if (url.indexOf(domain) !== -1) {
+        this.collector.addError(messages.RESTRICTED_HOMEPAGE_URL);
+      }
+      this.isValid = false;
+    });
```

Comment:
I would slightly prefer `for (let restrictedUrl of RESTRICTED_HOMEPAGE_URLS) {...}` and return earlier if we did already found that we are adding the error.

Also this.isValid should be set to false inside the if block, because we should only mark the manifest as invalid if the homepage url was actually a forbidden one. 

The unit tests should also catch this, and so let's make sure that we do have coverage to catch that mistake (a test case to verify that a non forbidden homepage url should not be marked as invalid).